### DISCUSSION TITLE
Dashboard support user data filter

### DISCUSSION THREAD

**Discussion by Cricle** [elsa-workflows/elsa-core#3395] [Q&A]
Can it support like saas, user mode data filter?

Like Azure, Google Cloud...

**Comment by sfmskywalker** [comment]
Elsa 2 doesn’t have this, but for Elsa 3 I’d like to build a modular dashboard app that might offer this type of functionality, out of the box or using custom modules. At the moment Elsa doesn’t have anything for this, so you might need to build a dashboard yourself or perhaps use a framework like Orchard Core or ABP and integrate Elsa into it.
